rend3-routine/shaders/src/structures_object.wgsl

```diff
@@ -7,17 +7,24 @@ struct Object {
     index_count: u32,
     material_index: u32,
     vertex_attribute_start_offsets: array<u32, {{vertex_array_counts}}>,
+    // 1 if enabled, 0 if disabled
+    enabled: u32,
 }
 
 struct ObjectRange {
     invocation_start: u32,
     invocation_end: u32,
     object_id: u32,
+    region_id: u32,
+    region_base_invocation: u32,
+    local_region_id: u32,
+    previous_global_invocation: u32,
+    atomic_capable: u32,
 }
 
 struct BatchData {
-    ranges: array<ObjectRange, 256>,
     total_objects: u32,
     total_invocations: u32,
     base_output_invocation: u32,
+    ranges: array<ObjectRange, 256>,
```

Comment:
There's no actual reason that we only have up to 256 objects in each batch. We have a limit of 256 in each region, but not within the batch.

Reply:
This is more complicated than it's currently worth.